src/components/CreateJobPage/CreateJobPage.js

```diff
@@ -41,6 +42,18 @@ const CreateJobPage = ({
   )
   const [templates, setTemplates] = useState([])
   const [showPanel, setShowPanel] = useState(false)
+  const history = useHistory()
+
+  useEffect(() => {
+    if (
```

Comment:
move it to a separate function. And also what if projectsNames.data.length === 0 ? Maybe we need to fetch them in this case